src/config/dynamic-theme-fixes.config

```diff
@@ -99,6 +99,15 @@ INVERT
 
 ================================
 
+*.crm*.dynamics.com
```

Comment:
Does this URL pattern work?

Reply:
This URL pattern worked when I tested it with the dark reader dev tools.